test/python/pulse/test_schedule.py

```diff
@@ -652,21 +653,27 @@ def test_filter_inst_types(self):
         for _, inst in no_pulse_and_fc.instructions:
             self.assertFalse(isinstance(inst, (Play, ShiftPhase)))
         self.assertEqual(len(only_pulse_and_fc.instructions), 4)
-        self.assertEqual(len(no_pulse_and_fc.instructions), 4)
+        self.assertEqual(len(no_pulse_and_fc.instructions), 5)
 
         # test on ShiftPhase
         only_fc, no_fc = \
             self._filter_and_test_consistency(sched, instruction_types={ShiftPhase})
         self.assertEqual(len(only_fc.instructions), 1)
-        self.assertEqual(len(no_fc.instructions), 7)
+        self.assertEqual(len(no_fc.instructions), 8)
+
+        # test on SetPhase
+        only_fc, no_fc = \
```

Comment:
should probably be called `only_setp, no_setp`

Reply:
Thank you! Fixed in https://github.com/Qiskit/qiskit-terra/pull/4540/commits/d1d776d5b32ff6a6e20e5f38fef2173375b607fe.